clang/test/SemaSYCL/stall_enable_host.cpp

```diff
@@ -0,0 +1,46 @@
+// RUN: %clang_cc1 -fsycl-is-host -fsyntax-only -ast-dump -Wno-sycl-2017-compat %s | FileCheck %s
```

Comment:
I personally don't think we need a separate host AST test here since we don't really care about the host. We have the CodeGen test already which verifies the attributes is ignored on host without diagnostics. In my opinion that is enough.